Makefile

```diff
@@ -15,9 +15,15 @@ URLS := \
     http://download.qemu-project.org/qemu-$(QEMU_VERSION).tar.xz.sig \
     https://kernel.org/pub/linux/kernel/v$(firstword $(subst ., ,$(LINUX_VERSION))).x/linux-$(LINUX_VERSION).tar.sign
 
-ALL_FILES := $(addprefix dl/,$(notdir $(patsubst %.sign,%.xz,$(patsubst %.sig,%,$(URLS))) $(filter %.sig, $(URLS)) $(filter %.sign, $(URLS))))
+ALL_FILES_TMP := $(notdir $(patsubst %.sign,%.xz,$(patsubst %.sig,%,$(URLS))) $(filter %.sig, $(URLS)) $(filter %.sign, $(URLS)))
```

Comment:
It looks like it could be specified as `$(notdir $(ALL_URLS))`. Am I missing something?

Reply:
You're right. We could simplify a bit more.